perun-ldapc/src/main/java/cz/metacentrum/perun/ldapc/processor/impl/FacilityAttributeProcessor.java

```diff
@@ -86,4 +91,24 @@ public void processAllAttributesRemoved(String msg, MessageBeans beans) {
 		}
 	}
 
-}
+	public void processVirtualAttributeChanged(String msg, MessageBeans beans) {
+		if (beans.getUser() == null || beans.getAttribute() == null || beans.getResource() == null) {
+			return;
+		}
+		try {
+			log.debug("Getting resources assigned to facility {}", beans.getFacility().getId());
+			List<Resource> resources = ldapcManager.getPerunBl().getFacilitiesManager().getAssignedResources(ldapcManager.getPerunSession(), beans.getFacility());
+
+			Attribute virtAttr = ((PerunBl) ldapcManager.getPerunBl()).getAttributesManagerBl().
+				getAttribute(ldapcManager.getPerunSession(), beans.getUser(), beans.getAttribute().getName());
```

Comment:
Why we get user attributes from core, if they are supposed to be facility attributes?